prisma/migrations/20231228133822_added_users_model/migration.sql

```diff
@@ -0,0 +1,20 @@
+-- AlterTable
+ALTER TABLE "Event" ADD COLUMN     "requires_payment" BOOLEAN NOT NULL DEFAULT true;
```

Comment:
pwede pa remove ng lines 1 and 2, cause mag error ang migration cause part eto ng iba na migration